### ISSUE TITLE
Encountered unresolved custom op: TensorListFromTensor.Node number 1118 (TensorListFromTensor) failed to prepare

### ISSUE THREAD

**Issue by ysl-cpp** [zqlsnr/SKIP-DPCRN-Base-Tflite-And-Onnx#1]
hello  zql:
       when i use the tflite model to denoise my wav: the python file :real_time_tflite.py ,it occures:
RuntimeError: Encountered unresolved custom op: TensorListFromTensor.Node number 1118 (TensorListFromTensor) failed to prepare.
Can you tell me the reason.  thank you very much!

**Comment by zqlsnr** [comment]
@ysl-cpp Hi, maybe I didn't set “converter.experimental_new_converter=True” When converting the tf-model， but it can run in tflite_runtime==2.5.0.

**Comment by ysl-cpp** [comment]
Yes, thank you very much and it has been sovled.By the way , the "output_details[3]['index']"  does not match the output index , I changed it to "output_details[0]['index']" and it works . I think it maybe the difference of tflite_runtime.interpreter  version

**Comment by ysl-cpp** [comment]
Thank you very much!
It has been solved。




余绍龙
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zqlsnr/SKIP-DPCRN-Base-Tflite-And-Onnx"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年12月7日(星期三) 下午3:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zqlsnr/SKIP-DPCRN-Base-Tflite-And-Onnx] Encountered unresolved custom op: TensorListFromTensor.Node number 1118 (TensorListFromTensor) failed to prepare (Issue #1)





 
Closed #1 as completed.
